packages/react/src/listbox-item/ListboxItem.tsx

```diff
@@ -31,11 +31,12 @@ export type ListboxItemBaseProps<
   >
 >;
 
-export const ListboxItemBase = forwardRef<HTMLDivElement, ListboxItemBaseProps>(
+export const ListboxItem = forwardRef<HTMLDivElement, ListboxItemProps>(
```

Comment:
Vai, do we need to set `role="option"` for the `Item` component?

Reply:
no, technically it will have `role="option"` or `role="menuitem"` depending on where it's used and when combined with another component that provides the behavior - but this component is only for appearance purposes and doesn't provide any behavior